content/actions/security-for-github-actions/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services.md

```diff
@@ -148,7 +148,7 @@ jobs:
       - name: Git clone the repository
         uses: {% data reusables.actions.action-checkout %}
       - name: configure aws credentials
-        uses: aws-actions/configure-aws-credentials@d4e5f6a7b8c9d0e1f2a3b4c5d6e7f8a9b0c1d2e3
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
I copied the commit hash from https://github.com/aws-actions/configure-aws-credentials/releases/tag/v4.

```suggestion
        uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502
```